### PR TITLE
some minor updates to plugin getting started tutorial docs

### PR DESCRIPTION
**Pull request by nmattis**
* [X] Your changes are not possible to do through a plugin and relevant
    to a large audience (ideally all users of OctoPrint)
  * [X] If your changes are large or otherwise disruptive: You have
    made sure your changes don't interfere with current development by
    talking it through with the maintainers, e.g. through a
    Brainstorming ticket
  * [X] Your PR targets OctoPrint's `devel` branch if it's a completely
    new feature, or `maintenance` if it's a bug fix or improvement of
    existing functionality for the current stable version (no PRs
    against `master` or anything else please)
  * [X] Your PR was opened from a custom branch on your repository
    (no PRs from your version of `master`, `maintenance`, or `devel`
    please), e.g. `dev/my_new_feature` or `fix/my_bugfix`
  * [X] Your PR only contains relevant changes: no unrelated files,
    no dead code, ideally only one commit - rebase and squash your PR
    if necessary!
  * [X] Your changes follow the existing coding style
  * [x] If your changes include style sheets: You have modified the
    `.less` source files, not the `.css` files (those are generated
    with `lessc`)
  * [x] You have tested your changes (please state how!) - ideally you
    have added unit tests
  * [x] You have run the existing unit tests against your changes and
    nothing broke
  * [ ] You have added yourself to the `AUTHORS.md` file :)

#### What does this PR do and why is it necessary?
Fixes command issue and updates other sections of plugin getting started documentation that was out of date

#### How was it tested? How can it be tested by the reviewer?
I ran through the tutorial

### CODE REVIEW

File: docs/plugins/gettingstarted.rst [hunk 1]
```diff
@@ -96,15 +96,15 @@ Apart from being discovered by OctoPrint, our plugin does nothing yet. We want t
 
    import octoprint.plugin
 
-   class HelloWorldPlugin(octoprint.plugin.StartupPlugin):
+   class HelloworldPlugin(octoprint.plugin.StartupPlugin):
```
Comment (cp2004) [inline docs/plugins/gettingstarted.rst:99]: 'Hello World' is two words, so it should stay as CamelCase for the class name.

**Review by cp2004 — CHANGES_REQUESTED** [review]
One tiny comment, otherwise this is OK.

File: docs/plugins/gettingstarted.rst (hunk 1 above)
Comment (nmattis) [reply docs/plugins/gettingstarted.rst]: Fixed 👍

**Review by cp2004 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**